LICENSE.txt

```diff
@@ -1,3 +1,21 @@
+  Some content is kindly provided by the open source community:
+
+     1. JmxScraper provided under Apache 2.0 License
+
+        https://github.com/prometheus/jmx_exporter
+
+        Copyright 2012-2015 The Prometheus Authors
+
+        Full License at
+        https://github.com/prometheus/jmx_exporter/blob/main/LICENSE
+
+        Compiled binary from
+        https://mvnrepository.com/artifact/io.prometheus.jmx/collector
+
+
+   ====================================================================
+   All other content is subject to the following license:
+
```

Comment:
IMHO, more appropriate to put into a NOTICE.txt file (similar to how the ASF projects do it)